Documents/Users/FeatureList.md

```diff
@@ -43,6 +43,11 @@ you can specify following line in .xvimrc.
 
      inoremap <C-e> <C-o>$
 
+## Navigation
+C-], C-t
+
+C-] is mapped to "Jump To Definition".
```

Comment:
So where is the Ctrl-] mapping implementation. I can't find changes corresponds to this.